apps/web/app/scripts/fetch-posts.js

```diff
@@ -0,0 +1,97 @@
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+/**
+ * Fetches all posts from the CMS by iterating through all pages
+ */
+async function fetchAllPosts() {
+  const allPosts = [];
+  let currentPage = 1;
+  let hasNextPage = true;
+  
+  console.log('Fetching posts from CMS...');
+  
+  while (hasNextPage) {
+    try {
+      console.log(`Fetching page ${currentPage}...`);
+      const response = await fetch(`https://cms.leather.io/posts?page=${currentPage}`);
```

Comment:
Why paginate the requests if we plan to use this to generate a `json` file with all f the responses? I think better to create an endpoint with one response for everything.

Reply:
The request is no longer paginated in latest commit